packages/@aws-cdk/aws-codebuild/test/test.codebuild.ts

```diff
@@ -581,6 +581,7 @@ export = {
             codebuild.FilterGroup.inEventOf(
               codebuild.EventAction.PULL_REQUEST_CREATED,
               codebuild.EventAction.PULL_REQUEST_UPDATED,
+              codebuild.EventAction.PULL_REQUEST_MERGED
```

Comment:
Leave the trailing comma here (look how beautiful the diff is because of it 😊)

Reply:
trailing comma added, committed, and pushed,

Reply:
I see what you did there... 😉